src/stores/game.js

```diff
@@ -378,6 +383,9 @@ export const useGameStore = defineStore('game', {
             data,
           },
           (res, jwres) => {
+            if (usingGameStateAPI && res === 'Not Found') {
```

Comment:
It's pretty weird to statically check the raw string. There's a better way to do this, probably by either allow listing the api paths than have been converted and checking that first, or making functions to generate each api path based on the env to get the right path the first time.